doc/examples/applications/plot_human_mitosis.py

```diff
@@ -0,0 +1,176 @@
+# hence, cell population growth). By definition, it is the ratio of cells in
+# mitosis (over the total number of cells). To analyze the above image,
+# we are thus interested in two thresholds: one separating the cells from the
+# background, the other separating the dividing nuclei (brighter spots) from
+# the cytoplasm of their respective mother cells (and, intensity-wise, from
+# the other cells). To separate these three different classes of pixels, we
+# resort to :ref:`sphx_glr_auto_examples_segmentation_plot_multiotsu.py`.
+
+thresholds = filters.threshold_multiotsu(image)
+regions = np.digitize(image, bins=thresholds)
+
+fig, ax = plt.subplots(ncols=2, figsize=(10, 5))
+ax[0].imshow(image)
+ax[0].set_title('Original')
+ax[0].axis('off')
+ax[1].imshow(regions)
+ax[1].set_title('Multi-Otsu thresholding')
+ax[1].axis('off')
+plt.show()
+
+#####################################################################
+# Since there are touching cells, thresholding is not enough to segment all
+# the cells. If it were, we could readily compute a mitotic index for this
+# sample:
+
+cells = image > thresholds[0]
+dividing = image > thresholds[1]
+labeled_cells = morphology.label(cells)
+labeled_dividing = morphology.label(dividing)
+naive_mi = labeled_dividing.max() / labeled_cells.max()
+print(naive_mi)
+
+#####################################################################
+# Whoa, this can't be! The number of dividing nuclei
+
+print(labeled_dividing.max())
+
+#####################################################################
+# is overestimated, while the total number of cells
+
+print(labeled_cells.max())
+
+#####################################################################
+# is underestimated.
+
+fig, ax = plt.subplots(ncols=3, figsize=(15, 5))
+ax[0].imshow(image)
+ax[0].set_title('Original')
+ax[0].axis('off')
+ax[2].imshow(cells)
+ax[2].set_title('Cells?')
+ax[2].axis('off')
+ax[1].imshow(dividing)
+ax[1].set_title('Dividing nuclei?')
+ax[1].axis('off')
+plt.show()
+
+#####################################################################
+# Count dividing nuclei
+# =====================
+# Clearly, not all connected regions in the middle plot are dividing nuclei.
+# On one hand, the second threshold (value of ``thresholds[1]``) appears to be
+# too low to separate those very bright areas corresponding to cell division
+# from relatively bright pixels otherwise present in many cells. On the other
+# hand, we want a smoother image, removing small spurious objects and,
+# possibly, merging clusters of neighbouring objects (some could correspond to
+# two emerging nuclei in one mother cell). In a way, the segmentation
+# challenge we are facing with dividing nuclei is the opposite of that with
+# (touching) cells.
+
+#####################################################################
+# To find suitable values for thresholds and filtering parameters, we proceed
+# by dichotomy, visually and manually.
+
```

Comment:
Here an alternative solution would be to use Hysteresis thresholding to keep connected components with values higher than the high multi-otsu threshold, but only if they also have pixels of higher value than an other (higher) threshold. This is a classical method to remove small spurious objects but have instead large smooth objects
```
divinding = filters.apply_hysteresis_threshold(image, thresholds[1], 1.4 * thresholds[1]), [0.5], colors="yellow"
```
The results are not completely the same as with the current method, but not being a biologist I would not know which result is the best.